src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,104 @@
+"""Create a robot template for purpose of syncing subclassof relations.
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/92
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/363
+"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Any, Dict, List, Union
+
+import pandas as pd
+from oaklib import get_adapter
+from oaklib.interfaces.basic_ontology_interface import RELATIONSHIP
+from oaklib.types import CURIE, URI
+
+from src.scripts.utils import DEFAULT_PREFIXES_CSV, get_monarch_curies_converter, remove_angle_brackets
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+ONTOLOGY_DIR = SRC_DIR / 'ontology'
+DEFAULT_OUTPATH = ONTOLOGY_DIR / 'reports' / 'sync-subclassof.robot.template.tsv'
+DEFAULT_ONTOLOGY_DB_PATH = ONTOLOGY_DIR / 'tmp' / 'merged.db'
+ROBOT_ROW = [{
+    'term_id': 'ID',
+    'parent_id': 'SC % SPLIT=|',
```

Comment:
## `robot` subheader
How's this?
Showing first 3 rows:
|term_id      |parent_id   |xref_ids                                                                    |
|-------------|------------|----------------------------------------------------------------------------|
|ID           |SC % SPLIT=&#124;|A oboInOwl:hasDbXref SPLIT=&#124;                                                |
|MONDO:0000001|BFO:0000016 |Orphanet:377788&#124;NCIT:C2991&#124;DOID:4&#124;UMLS2:C0012634&#124;SCTID:64572001&#124;MESH:D004194|

Reply:
Ah OK. I remembered you said that, and I was actually going to bring it up, but for some reason thought best not to bug you about it / that there were OK exceptions to this. Seems like I was wrong! I will filter.

Reply:
FYI there were only 3 exceptions to this. Anyway this is done now.